api/mysagw/case/views.py

```diff
@@ -208,13 +209,17 @@ def merge(self, name, data, template, file_name):
     @action(detail=True)
     def application(self, request, pk=None):
         caluma_client = self.get_caluma_client(request)
-        raw_data = caluma_client.get_data(pk, GQL_DIR / "get_document.gql")
+        language = get_language()
+        raw_data = caluma_client.get_data(
+            pk,
+            GQL_DIR / "get_document.gql",
+            add_headers={"Accept-Language": language},
+        )
```

Comment:
Nice to get rid of code. I understand that reading the document in one's preferred language (the user exporting the application). The user entered text is not translated, though that would provide a mixed-language document (questions this, answers that), which need the reviewer to be bilingual. Is that intended? The resulting pdf in case of differing languages also is not the version that was "handed in" by the applicant

Reply:
This way the user gets the same data exported as they see in the UI. Differing languages will only happen for multilang teams and they should know, the form could be filled in a different language.

This will be part of a thorough review by the customer and might be subject to change.